tests/conftest.py

```diff
@@ -15,53 +16,58 @@
 
 # read in SQL for populating test data
 with open(os.path.join(os.path.dirname(__file__), "data.sql"), "rb") as f:
-    data_sql = f.read().decode("utf8")
+	data_sql = f.read().decode("utf8")
 
 @pytest.fixture
 def app():
-    # create a temporary database's path in memory.
-    # db_fd is file descriptor and db_path is database path
-    # this line will be removed in the future after we have request message model
-    # and use database file on disk for testing
-    db_fd, db_path = tempfile.mkstemp()
-
-    app = create_app({
-        'TESTING': True,
-        'SQLALCHEMY_DATABASE_URI': TEST_DB_URI,
-        'DATABASE': db_path, # this line will be removed like db_df, db_path
-    }, __name__)
-
-    with app.app_context():
-        get_db().executescript(data_sql) # this line will be removed like db_df, db_path
-        yield app
-
-    # close and delete database file and path after the test is over
-    # these lines will be removed like db_db, db_path
-    os.close(db_fd)
-    os.unlink(db_path)
+	# create a temporary database's path in memory.
+	# db_fd is file descriptor and db_path is database path
+	# this line will be removed in the future after we have request message model
+	# and use database file on disk for testing
+	db_fd, db_path = tempfile.mkstemp()
+
+	app = create_app({
+		'TESTING': True,
+		'SQLALCHEMY_DATABASE_URI': TEST_DB_URI,
+		'DATABASE': db_path, # this line will be removed like db_df, db_path
+	}, __name__)
+
+	with app.app_context():
+		get_db().executescript(data_sql) # this line will be removed like db_df, db_path
+		yield app
+
+	# close and delete database file and path after the test is over
+	# these lines will be removed like db_db, db_path
+	os.close(db_fd)
+	os.unlink(db_path)
```

Comment:
As we have discussed that we will create the database test in memory, should we make changes to these comments?

Reply:
Do you mean the database is in the ram instead of in our hard drive? So I should change my test for the user's database too (The test for the user's model creates a database on the hard drive and delete it after finishing testing)? Sorry I missed the last meeting.

Reply:
Yes, we have decided that it is okay to have the test database now created in memory and will be removed from memory when tests finish. Luc's PR has implemented this but still leaves the comments for the old decision.

In the future, if we have more tests, we might need to create the test database file on disk.

Reply:
If creating the test database in memory is a temporary solution and will be changed after having more tests, we should keep those comments.